plugins/wasm-rust/src/event_stream.rs

```diff
@@ -86,6 +77,24 @@ impl EventStream {
 
         None
     }
+}
+impl EventStream {
+    pub fn new() -> Self {
```

Comment:
There is no need to keep new but just use default to construct, and I don't think it's a good habit that mixing different things in single PR

Reply:
done, I made modifications only after discovering an error with clippy during local testing